clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -3108,15 +3180,25 @@ public void testAbortTransactionAndResetSequenceNumberOnUnknownProducerId() thro
         transactionManager.beginTransaction();
         transactionManager.maybeAddPartition(tp0);
 
-        prepareAddPartitionsToTxnResponse(Errors.NONE, tp0, epoch, producerId);
+        if (!transactionV2Enabled) {
+            prepareAddPartitionsToTxnResponse(Errors.NONE, tp0, epoch, producerId);
+        } else {
+            prepareAddPartitionsToTxnResponse(Errors.NONE, tp0, (short) (epoch + 1), producerId);
+        }
         runUntil(() -> transactionManager.isPartitionAdded(tp0));
 
         assertEquals(0, transactionManager.sequenceNumber(tp0));
-        assertEquals(1, transactionManager.sequenceNumber(tp1));
+
```

Comment:
nit: we don't necessarily need to change this test here, but it was originally checking the sequence is reset on tp0 when it saw the error. Now we will always start with sequence 0 after aborting, so the end result is not as substantial. I think it is useful to go through the abort flow here though.

Reply:
The reason why I modified it was because when we use Tv2 the sequence is reset for tp1 on abort too, I think nothing really changes for tp0 like you mentioned.

Reply:
Right, I was just saying that the check for the sequence is not the main point in the test for tv2. We could do something like check epoch is bumped for example. For TV 0 it is pretty important.

Reply:
The epoch bump in this case (unknown pid Error) is being checked in another test, discussed offline to remove the V2 check for when initPid version doesn't support epoch bump